crates/typos-cli/src/file_type_specifics.rs

```diff
@@ -7,6 +7,15 @@
 pub const NO_CHECK_TYPES: &[&str] = &["cert", "lock"];
 
 pub const TYPE_SPECIFIC_DICTS: &[(&str, StaticDictConfig)] = &[
+    (
+        "css",
+        StaticDictConfig {
+            ignore_idents: &[
+                "nd", // CSS class used by pygments (see https://github.com/pygments/pygments/blob/2.16.1/pygments/token.py#L146)
```

Comment:
This is feeling quite specialized for us to include css classes from specific generators.  People can always put this in their own config

Reply:
CSS files for Pygment styles are ubiquitous. The GitHub code search [reports 10.1k files](https://github.com/search?q=Name.Decorator+language%3ACSS+.nd&type=code).

Reply:
Eh, if its only css